src/main/java/bhs/devilbotz/commands/DriveSetDistancePID.java

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package bhs.devilbotz.commands;
+
+import bhs.devilbotz.subsystems.DriveTrain;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class DriveSetDistancePID extends CommandBase {
+  private DriveTrain drivetrain;
+  private PIDController distance_pid;
+  private double distance;
+
+  public DriveSetDistancePID(DriveTrain drivetrain, double distance) {
+    this.drivetrain = drivetrain;
+    this.distance = distance;
+    distance_pid = new PIDController(0.25, 0, 0.1);
```

Comment:
In the final code, we should put these as constants elsewhere so that we can use the same code on different robots easily.